src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (UserService.findByEmail(email) != null) {
+            if (UserService.findByEmail(email).getPassword().equals(password)) {
```

Comment:
Also consider to fetch user by email once at the beginning. Following this, you won't need to call `findByEmail` method each time you need to conduct some checks.